src/components/specific/app/app-header-menu/language-selector/LanguageSelector.scss

```diff
@@ -12,6 +12,10 @@
     align-items: center;
     margin-bottom: var(--spacing-unit);
     font-weight: bold;
+
+    span {
```

Comment:
@LrxGaelle C'est OK ça ou on prefère utiliser une autre méthode?

Reply:
We can use `class="m-l-6"` in HTML template instead of `margin-left: calc(var(--spacing-unit) / 2);` in style